geode-redis/README.md

```diff
@@ -128,210 +128,98 @@ not connected>
-|                    	|                                                    	| ZRANGEBYSCORE                 	|
-|                    	|                                                    	| ZRANK                         	|
-|                    	|                                                    	| ZREM                          	|
-|                    	|                                                    	| ZREMRANGEBYLEX                	|
-|                    	|                                                    	| ZREMRANGEBYRANK               	|
-|                    	|                                                    	| ZREMRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANGE                     	|
-|                    	|                                                    	| ZREVRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANK                      	|
-|                    	|                                                    	| ZSCAN                         	|
-|                    	|                                                    	| ZSCORE                        	|
-|                    	|                                                    	| ZUNIONSTORE                   	|                   	|
-
-**NOTES:**
-
-[1] - UNLINK is implemented as a synonym to DEL and does not unlink asynchronously.
-[2] - SLOWLOG is implemented as a NoOp.
+The Geode APIs compatible with Redis implement a subset of the full Redis command set.
+
+#### <a name="supported-redis-commands"></a> Supported Redis Commands [Return to top](#introduction)
+- APPEND 
+- AUTH
+- DECR
+- DECRBY
+- DEL
+- EXISTS
+- EXPIRE
+- EXPIREAT
+- GET
+- GETRANGE
+- HDEL
+- HEXISTS
+- HGET
+- HGETALL
+- HINCRBY
+- HINCRBYFLOAT
+- HLEN
+- HMGET
+- HMSET
+- HSCAN
+- HSET
+- HSETNX
+- HSTRLEN
+- HVALS
+- HKEYS
+- INCR
+- INCRBY
+- INCRBYFLOAT
+- INFO <sup>1</sup>
+- KEYS
+- MGET
+- PERSIST
+- PEXPIRE
+- PEXPIREAT
+- PING
+- PSUBSCRIBE
+- PTTL
+- PUBLISH
+- PUNSUBSCRIBE
+- QUIT
+- RENAME
+- SADD
+- SET
+- SETNX
+- SLOWLOG <sup>2</sup>
+- SMEMBERS
+- SREM
+- STRLEN
+- SUBSCRIBE
+- TTL
+- TYPE
+- UNSUBSCRIBE
+
+**NOTES:** <br/>
+<sup>1</sup> INFO is implemented for the sections and fields listed below:
+ - clients
+    - connected_clients
+    - blocked_clients (will always return 0)
+ - cluster
+    - cluster_enables (will always return 0)
+ - keyspace
+    - returns stats for db: 0
+ - memory
+    - maxmemory
+    - used_memory
+    - mem_fragmentation_ratio (will always report 1.00) 
+ - persistence
+    - loading (will always return 0)
+    - rdb_changes_since_last_save (will always return 0)
+    - rdb_last_save_time (will always return 0)
+ - replication
+    - role
+    - connected_slaves (will always return 0)
+ - server
+   - redis_version
+   - redis_mode
```

Comment:
This is always `standalone` right now... not sure if we need to say that or not?

Reply:
yeah, I wasn't sure how far to go with saying what is hardcoded, but I guess it makes to be consistent and do it for all hard-coded values